vscode-dotnet-runtime-library/src/Acquisition/InstallTrackerSingleton.ts

```diff
@@ -120,7 +120,7 @@ export class InstallTrackerSingleton
 
     public async clearPromises() : Promise<void>
     {
-        await this.executeWithLock( false, () => {this.inProgressInstalls.clear();});
+        this.inProgressInstalls.clear();
```

Comment:
I noticed quite a few functions in this file which are async but have no async code inside it like this one.

Reply:
ah yes, good point. these no longer need to be async with this change.